x-pack/legacy/plugins/console_extensions/spec/generated/ml.put_trained_model.json

```diff
@@ -0,0 +1,11 @@
+{
+  "ml.put_trained_model": {
+    "methods": [
+      "PUT"
+    ],
+    "patterns": [
+      "_ml/inference/{model_id}"
+    ],
+    "documentation": "TODO"
```

Comment:
Can we have the link to the doc?

Reply:
Yeah, couldn't see that there was a page for this so I just linked to ML overview page inside `x-pack/legacy/plugins/console_extensions/spec/overrides/ml.put_trained_model.json`. Not sure what else to do for now 😅